lib/beaker/options/presets.rb

```diff
@@ -110,7 +110,7 @@ def env_vars
       def presets
         h = Beaker::Options::OptionsHash.new
         h.merge({
-          :project                => 'Beaker',
+          :project                => ENV['JOB_NAME'] || 'Beaker',
```

Comment:
@waynr is correct - we try and pull in all the env vars in one chunk at one go so that it is easier to track what variables are supported.